addOns/sequence/CHANGELOG.md

```diff
@@ -5,8 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Added
-- Add `sequence-import` Automation Framework job.
-- Initial sequence-activeScan implementation.
+- Add Automation Framework jobs:
```

Comment:
The heading is added. I think you can just start with "Automation..."

Reply:
I know you probably just rolled with what was there earlier :wink:

Reply:
No rolls, that was on purpose, previous line was me too :)